src/helpers/layout.js

```diff
@@ -5,6 +5,7 @@ class LayoutHelper {
     constructor(ctx) {
         this.ctx = ctx;
         this.handlers = [];
+        this.ctx.addEventListener('mouseleave',()=>{this.hideMoreOptions();});
```

Comment:
Missing spaces. We don't have a prettifier, but let's use the typical styling guidelines (in accordance with the specific of this project).